assets/overhead_test/main.py

```
@@ -0,0 +1,81 @@
+        Runs the benchmarked tool
+        """
+        self.docker_execute('sleep 5')
+
+    def ingest(self):
+        """
+        Ingests the dataset at self.datasets_path
+        """
+        #print(self.datasets_path)
+        self.docker_execute('sleep 15')
+    
+    def search(self, query):
```

Ruff (0.12.2)

Missing return type annotation for public function `search`

Add return type annotation: `int`

(ANN201)